VM memory tests and swap prevention

ivan4th opened the pull request:
- Set memoryBacking/locked for domains
- Make `examples/virsh.sh` accept `@pod[:namespace]` notation
- Refactor `tests/e2e/e2e.sh`
- Add memory tests

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/mirantis/virtlet/278)
<!-- Reviewable:end -->

jellonek commented:
Reviewed 6 of 6 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/278)*
<!-- Sent from Reviewable.io -->

vefimova commented:
Reviewed 6 of 6 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[examples/virsh.sh, line 26 at r1](https://reviewable.io:443/reviews/mirantis/virtlet/278#-KkFmy7GQx_up4aiIxt9:-KkFmy7GQx_up4aiIxtA:b-et77s5) ([raw file](https://github.com/mirantis/virtlet/blob/8d0c6d66e1235367ce291df2c5bc70f0b87ff40d/examples/virsh.sh#L26)):*
> ```Shell
>   local namespace_opts=""
>   if [[ ${namespace} ]]; then
>       namespace_opts="-o ${namespace}"
> ```

should be "-n" option here, no?

---

*[tests/e2e/e2e.sh, line 43 at r1](https://reviewable.io:443/reviews/mirantis/virtlet/278#-KkFRm0fAQ6lkQ6EpYdX:-KkFRm0giBkCd9D7mj5y:bsnr462) ([raw file](https://github.com/mirantis/virtlet/blob/8d0c6d66e1235367ce291df2c5bc70f0b87ff40d/tests/e2e/e2e.sh#L43)):*
> ```Shell
>     fi
>     if ((--n == 0)); then
>       echo "Timed out waiting for pod ${pod}" >&2
> ```

"exit 1" missed?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/278)*
<!-- Sent from Reviewable.io -->

ivan4th commented:
Review status: 4 of 6 files reviewed at latest revision, 2 unresolved discussions.

---

*[examples/virsh.sh, line 26 at r1](https://reviewable.io:443/reviews/mirantis/virtlet/278#-KkFmy7GQx_up4aiIxt9:-KkG2DPx-wTv7pYTgl-N:bn5m0ab) ([raw file](https://github.com/mirantis/virtlet/blob/8d0c6d66e1235367ce291df2c5bc70f0b87ff40d/examples/virsh.sh#L26)):*
<details><summary><i>Previously, vefimova (Victoria Efimova) wrote…</i></summary><blockquote>

should be "-n" option here, no?
</blockquote></details>

Thanks for spotting this, fixed.

---

*[tests/e2e/e2e.sh, line 43 at r1](https://reviewable.io:443/reviews/mirantis/virtlet/278#-KkFRm0fAQ6lkQ6EpYdX:-KkG2AHpKocjORtNbSbj:bj3eodx) ([raw file](https://github.com/mirantis/virtlet/blob/8d0c6d66e1235367ce291df2c5bc70f0b87ff40d/tests/e2e/e2e.sh#L43)):*
<details><summary><i>Previously, vefimova (Victoria Efimova) wrote…</i></summary><blockquote>

"exit 1" missed?
</blockquote></details>

Fixed, thanks!

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/278)*
<!-- Sent from Reviewable.io -->

vefimova commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/278#-:-KkG3P9Z8Tbf5sjrF97S:bnfp4nl)*
<!-- Sent from Reviewable.io -->

jellonek commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/278#-:-KkGLtQoeZKN40dym7bk:bnfp4nl)*
<!-- Sent from Reviewable.io -->
